baus/earthquake.py

```diff
@@ -280,7 +274,7 @@ def earthquake_demolish(parcels, parcels_tract, tracts_earthquake, buildings,
         print("Number of census tract groups is: %d" % len(tract_parcels_grp))
 
         # for the parcels in each tract, destroy X% of parcels in that tract
-        tracts_earthquake = tracts_earthquake.to_frame()
+        tracts_earthquake = orca.get_table("tracts_earthquake").to_frame()
```

Comment:
Good question. I think I did this to make it a bit more self-contained, meaning if this code section is run, then read in the table... and now I'm wondering if it was also because that also makes having the file in your inputs/read in by `datasources.py` optional (whereas if it's an argument the code may need it to exist).